src/main/java/labs/introtoprogramming/lab5/object/Builder/LoadDataFromParserObject.java

```diff
@@ -0,0 +1,431 @@
+package labs.introtoprogramming.lab5.object.Builder;
+
+import labs.introtoprogramming.lab5.geometry.Vector2;
+import labs.introtoprogramming.lab5.geometry.Vector3;
+import labs.introtoprogramming.lab5.object.Builder.POJO.FacePlate;
+import labs.introtoprogramming.lab5.object.Builder.POJO.Material;
+import labs.introtoprogramming.lab5.object.Builder.POJO.ReflectivityTransmission;
+import labs.introtoprogramming.lab5.object.Builder.POJO.VertexFace;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.INFO;
+
+public class LoadDataFromParserObject implements BuilderInterface {
```

Comment:
Why is this class separate from parser? If you want to load info from file than create child of SceneObject and load data to it. The idea was that info about material, face, texture maps are stored in Mesh object. So it is better to load all data in mesh.